.github/workflows/test.yaml

```diff
@@ -0,0 +1,25 @@
+name: mod landing page tests
+on:
+  pull_request:
+    branches:
+      - blue_team_code
+  push:
+    branches:
+      - gh_action-mod-pages-tests
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+      - run: git fetch --no-tags --prune --depth=5 origin ${{ github.head_ref }}
+      - uses: actions/setup-node@v2
+        name: setup node
+        with:
+          node-version: 12
+      - run: npm install
+      - run: npm run test
```

Comment:
oh weird that this just works without installing nx? The readme states that we have to isntall nx, which I thought was what my nx step was doing. But `npm run test` *does* seem to be running `nx test` and it works. So let's do it

Reply:
Yep, lets try it and see. Previously it did not work, so harm no foul. :-)